config/crd/bases/osp-director.openstack.org_openstackcontrolplanes.yaml

```diff
@@ -83,6 +83,9 @@ spec:
                 description: GitSecret used to pull playbooks into the openstackclient
                   pod
                 type: string
+              idmSecret:
```

Comment:
we should have a tls-e chapter in the Readme doc and explain how the secret content needs to look like, or have a separate doc file for tls-e?

Reply:
+1 will followup with docs for TLSe and TLS public endpoints

Reply:
sounds good thanks